src/main/java/io/leangen/graphql/GraphQLSchemaGenerator.java

```diff
@@ -747,7 +764,7 @@ private void applyProcessors(GraphQLSchema.Builder builder) {
     }
 
     private boolean isInternalType(GraphQLType type) {
-        return type.getName().startsWith("__") || type.getName().equals(QUERY_ROOT) || type.getName().equals(MUTATION_ROOT);
+        return type.getName().startsWith("__") || type.getName().equals(queryRoot) || type.getName().equals(mutationRoot);
```

Comment:
Please add `|| type.getName().equals(subscriptionRoot)`.
I know it's unrelated to the PR but it's a bug that it's not there.

Reply:
Done.